### PR TITLE
Fix style rule editor

### PR DESCRIPTION
**Pull request by xithan**
Based on #1017 because of the Cypress upgrade that is included there.
Fixes only one Coffeescript file and adds a Cypress tests

### CODE REVIEW

**Review by ethn — COMMENTED** [review]
It's quite possible I'm being thick, but I don't actually see any code changes that seem like they could fix the style rule editor (except, I guess, gem updates?). I see some good tests and some styling fixes. 

What fixed things?

**Comment by xithan** [comment]
The `window.ready` in `click_select.js.coffee` is the fix

**Comment by xithan** [comment]
I don't understand the broken cypress tests. It looks like the four failing tests have no css.

**Comment by xithan** [comment]
I have a theory now: Maybe the "getting start" cyp breaks the css.

File: SEEDME.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ core
 
 This tells you:
 
-1. Running `be decko setup` will use fixtures from the **defaults** mod, the first mod on
+1. Running `bundle execute decko setup` will use fixtures from the **defaults** mod, the first mod on
```
Comment (ethn) [inline SEEDME.md:67]: exec, not execute.

**Review by ethn — APPROVED** [review]

**Pull request merged**